app/models/state_file_dependent.rb

```diff
@@ -57,4 +57,11 @@ def ask_senior_questions?
   def self.senior_cutoff_date
     MultiTenantService.statefile.end_of_current_tax_year.years_ago(65)
   end
+
+  def age
+    # Below is age as of today
+    ((Time.now.beginning_of_day - dob.to_time) / 1.year.seconds).floor
+    # Below is age as of the end of the year
+    # DateTime.current.year - dob.time.year
```

Comment:
Based on [this slack thread](https://cfastaff.slack.com/archives/C0544ERAFQV/p1703198025859729) I was expecting this math to use `end_of_current_tax_year` instead of the current day.

Reply:
You're right - Updated

Reply:
Nice! The test cases should be updated too - looks like there are some failures